app/views/admin/versions/_event_change_description.html.haml

```diff
@@ -0,0 +1,35 @@
+- if version.event == 'create'
```

Comment:
Please use complete sentences when communicating. It helps understanding what you are trying to say very much :-)